ddtrace/tracer/tracer.go

```diff
@@ -320,14 +379,19 @@ func (t *tracer) flushPayload() {
 	if t.payload.itemCount() == 0 {
 		return
 	}
+	defer func(start time.Time) {
+		t.config.statsd.Timing("datadog.tracer.flush.duration", time.Since(start), nil, 1)
+	}(time.Now())
 	size, count := t.payload.size(), t.payload.itemCount()
 	log.Debug("Sending payload: size: %d traces: %d\n", size, count)
 	rc, err := t.config.transport.send(t.payload)
 	if err != nil {
+		t.config.statsd.Count("datadog.tracer.flush.traces_lost", int64(count), nil, 1)
```

Comment:
There's too many metrics that mean the same thing. All lost traces should be in the same metric: `datadog.tracer.traces_dropped`, but with different tags. For example: `reason:flush_error` and `reason:buffer_full`